### ISSUE TITLE
Show dialog for backend errors in Angular

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#16199]
As a stakeholder I want to make sure that developers can send notifications to the user from the backend.

## The Problem
Backend developers needs to send information to the frontend, we have the toast for small messages and warning but we need something bigger for when this messages (or errors) carry A LOT of content like:
1. Stacktraces
2. Code (a lot of it)
3. Actions, link, etc...

## Solution
We're going to listen for a specific `Websocket` event that, for now will be an object with the following props:

* Title: string
* Width: number
* Content: string

### The Content
The content could be anything, code, html, stacktrace, it could contain links, table, list, etc... and, in the future even Javascript (we're not covering that right now but we have to keep it in mind)

Basically, it could be anything.

### The Dialog
We need to show this content using the `dot-dialog` component. Show the title prop as the dialog title, if a width come set it as the dialog width and the content in the body of the dialog

## Acceptance Criteria

1. Listen the `Websocket` event and display the props of the object in the `dot-dialog`
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by fmontes** [comment]
To use it we need to send from the websocket an event with the title `LARGE_MESSAGE` and the data should be:

```
export interface DotLargeMessageDisplayParams {
    title: string;
    width?: string;
    height?: string;
    body?: string;
    code?: {
        lang: string;
        content: string;
    };
}
```

**Comment by fmontes** [comment]
QA Feedback:

1. Create unit test for this: https://github.com/dotCMS/core-web/commit/e598aac61336652f42a1103765f87790b34ed466
2. Check why the dialog is not taking inner html of inline style to javascript
3. Set a default height and width

**Comment by jdotcms** [comment]
The message is being rendered ok.
Just couple notes, the message and the script should have html tag to be rendered.
If you are for instance on the content editor and execute a task that trigger a message, you have to close the content editor to see the dialog.

<img width="614" alt="largemessageactionlet" src="https://user-images.githubusercontent.com/19578350/56989054-be8c5600-6b4e-11e9-8afa-f2f0f5f82729.png">

<img width="1795" alt="Screen Shot 2019-04-30 at 1 46 52 PM" src="https://user-images.githubusercontent.com/19578350/56989060-c4823700-6b4e-11e9-91f7-01e23fef73af.png">

**Comment by bryanboza** [comment]
That does not make sense, If you are in a content editor and you fire any action with a message, we need to show this in front as a pop-up, right now you need to close the content editor to see the message

**Comment by wezell** [comment]
Fixes: #16403

**Comment by fmontes** [comment]
Now the message dialog shows on top of any other dialog in the system.

**Comment by jdotcms** [comment]
Now the Dialog is working better. However when you have several tabs open, render the dialog (will be rendered on both tab at the same time), close one of them, and render again, the second starts appending the body, for instance

<img width="669" alt="Screen Shot 2019-05-07 at 10 19 32 AM" src="https://user-images.githubusercontent.com/19578350/57316060-2fcd8b00-70b2-11e9-8131-61161d5c06c1.png">
<img width="755" alt="Screen Shot 2019-05-07 at 10 19 09 AM" src="https://user-images.githubusercontent.com/19578350/57316068-322fe500-70b2-11e9-9002-f6fc56d6f199.png">

**Comment by wezell** [comment]
When calling the large message window from java, we need to pass a dialog id.  This id will be used to identify the dialog in the ui when it is popped up on a user's screen.  If other messages are passed using the same id, they will be appended to the dialog div (tbd).  If a different id is passed, then another dialog will be popped up on top of the original dialog.

The purpose of this is to enable a dialog to work both when performing a workflow action on a single contentlet or when performing a workflow action across multiple contentlets - for example during a batch process.

**Comment by fmontes** [comment]
Ok, now if we send another event and there is opened dialog we popups another one on top. Use wisely.

**Comment by bryanboza** [comment]
After test in the last master we have a few problems here: 

- We are not showing the value of the "Message" field in the window 

- To close the message window you need to click 3 times the "X"

**Comment by bryanboza** [comment]
After the last fixed we are opening two windows, one without the body and the new one with the body content. 
SC: https://screencast.com/t/cAKepN3s

**Comment by alfredo-dotcms** [comment]
not able to reproduce this, with latest `master`
https://monosnap.com/file/Jed9iD1wM6xHV96ES96IToZ6edUq6k
